registry.json

```diff
@@ -4218,5 +4218,19 @@
     "telegram": "EarnCoinPool",
     "operatorName": "Rich",
     "github": ""
+  },
+  {
+    "ticker": "MERK",
+    "poolId": "232b0ab2239204974aa236cf022ab95301a1772609f3c6907628cde2",
+    "social": {
+        "website": "https://merklestaking.io",
+        "twitter": "MerkleValidator",
+        "facebook": "",
+        "instagram": "",
+        "youtube": ""
+    },
+    "telegram": "MerkleValidator",
```

Comment:
Please can you change this to the Telegram handle of an individual and not a channel or group?

Reply:
Updated to point to admin handle [dhruvthak](https://t.me/dhruvthak)